src/components/FilterByCentury/CenturyAllButon.tsx

```diff
@@ -0,0 +1,22 @@
+import classNames from 'classnames';
+import { SearchLink } from '../SearchLink/SearchLink';
+import { useSearchParams } from 'react-router-dom';
+
+export const CenturyAllButon = () => {
+  const [searchParams] = useSearchParams();
+  const centuries = searchParams.getAll('centuries');
+
+  return (
+    <div className="level-right ml-4">
+      <SearchLink
+        data-cy="centuryALL"
+        params={{ centuries: null }}
```

Comment:
Setting `params={{ centuries: null }}` might not work as expected if the `SearchLink` component does not handle `null` values correctly. Consider using an empty array or a more explicit method to clear the `centuries` parameter, depending on how `SearchLink` is implemented.